azdev/operations/help/refdoc/common/directives.py

```diff
@@ -63,23 +63,23 @@ class CliCommandDirective(CliBaseDirective):
         Field('docsource', label='Doc Source', has_arg=False,
               names=('docsource', 'documentsource')),
         Field('deprecated', label='Deprecated', has_arg=False,
-              names=('deprecated'))
+              names=('deprecated',))
     ])
 
 
 class CliArgumentDirective(CliBaseDirective):
     doc_field_types = copy.copy(_CLI_FIELD_TYPES)
     doc_field_types.extend([
         Field('required', label='Required', has_arg=False,
-              names=('required')),
+              names=('required',)),
```

Comment:
This changes the semantics of the code. Better to confirm if it is required or works as expected.

Reply:
`names` should be `Tuple[unicode, ...]`.
https://github.com/sphinx-doc/sphinx/blob/645044d7f03c7fcb4c39a8edf40aa66f6d6c2b64/sphinx/util/docfields.py#L55-L57

Why this type error never found?

Reply:
I comfirm this is a typo, as next line it becomes a tuple:
https://github.com/Azure/azure-cli-dev-tools/blob/cabbb3ef6eeb398a5100f14c19d5cc3f4a8ac840/azdev/operations/help/refdoc/common/directives.py#L75-L76